### ISSUE TITLE
XLRangeBase.cs not found in current version 0.87.1

### ISSUE THREAD

**Issue by janbro1** [ClosedXML/ClosedXML#397]
**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature

**Version of ClosedXML**
0.87.1

**What is the current behavior?**
I freshly installe the package from Nuget. The development machine has no direct access to the internet. 
I get a XLRangeBase.cs not found. When using following code (an example I took from the net, to ensure no typo on my end):
            ws.Cell(1, 1).Value = "Link to a web page, no tooltip - Yahoo!";
            ws.Cell(1, 1).Hyperlink = new XLHyperlink(@"http://www.yahoo.com");
after taking the previous version uninstall newer and installed older, no more issue.

**If the current behavior is a bug, please provide the steps to reproduce and 
if possible a minimal demo of the problem with a sample spreadsheet.**
no error shown during debug, but upon run, unhandled exception is thrown.

**What is the expected behavior or new feature?**
Adding of Hyperlink in chosen cell.
**Did this work in previous versions of our tool?  Which versions?**
I took 0.76.0 and - without adjustment to my code - ran through quite nicely.
- [ ] I attached a sample spreadsheet.  (You can drag files on to this issue)

**Code to reproduce problem:**
```c#
using ClosedXML.Excel;
public void Main()
{
            ws.Cell(1, 1).Value = "Link to a web page, no tooltip - Yahoo!";
            ws.Cell(1, 1).Hyperlink = new XLHyperlink(@"http://www.yahoo.com");
}
```

**Comment by igitur** [comment]
Your code sample is incomplete. I tried to infer the missing parts, but then I can't reproduce the error.

**Comment by igitur** [comment]
This is the code that I inferred. 
```c#
static void Main(string[] args)
{
    using (var wb = new XLWorkbook())
    {
        var ws = wb.Worksheets.Add("sheet1");
        ws.Cell(1, 1).Value = "Link to a web page, no tooltip - Yahoo!";
        ws.Cell(1, 1).Hyperlink = new XLHyperlink(@"http://www.yahoo.com");
    }
}
```

**Comment by janbro1** [comment]
In general you stated it correctly. I had:
 private XLWorkbook doSomething()
        {
            int i=1;
            var workbook = new XLWorkbook();
            var ws = workbook.Worksheets.Add("2Dos");
            ws.Cells("b" +i).Value = "text";
            ws.Cell(1, 1).Value = "Link to a web page, no tooltip - Yahoo!";
            ws.Cell(1, 1).Hyperlink = new XLHyperlink(@"http://www.yahoo.com");

        return workbook;
        }
        public void createExcelFile() {
            XLWorkbook workbook =doSomething();
            workbook.SaveAs("d:\\data\\MaS2Dos.xlsx");
        }

Again, this did not work with the current version, but when I installed the "older" version I had no issues.

**Comment by vbjay** [comment]
"b"+i.ToString()

**Comment by igitur** [comment]
@janbro1 Your code is still invalid. You don't declare the `workbook` variable anywhere. Give me a code sample that I can copy and paste into a *console application* and run directly.

**Comment by janbro1** [comment]
@all I think it is quite simple. My code covers now several hundred lines, which are 95% irrelevant.
If there was an issue with my code, it wouldn't work, but it does with an older Version as stated. From what I see it appears as if there is a reference missing in your package. 

Again, for the version 0.87.1 (I got from nuget as an offline package) the error XLRangeBase.cs. If I download and try 0.76.0 everything works nicely. Second thing the Error does not show until you execute.
I gave you the code that triggered the exception already. 

            ws.Cell(1, 1).Value = "Link to a web page, no tooltip - Yahoo!";
            ws.Cell(1, 1).Hyperlink = new XLHyperlink(@"http://www.yahoo.com");

And last thing, I totally understand and appreciate your help, but there is nothing else I can add to this. I have wasted way to much time already (needlessly assessing my code for a non-existing error) and than repeating myself here.
@igitur XLWorkbook workbook = new XLWorkbook (); the workbook is returned and than saved (in this case)
